### ISSUE TITLE
Missing accessibility labels for buttons at the bottom when reading articles

### ISSUE THREAD

**Issue by Neurrone** [wallabag/ios-app#137]
| Q             | A
| ------------- | ---
| Device?       | iPhone
| Device Os version    | 11.2.1
| App Version   | 2.0
| Wallabag server version | 2.0

I'm using the built-in VoiceOver screen reader on the iPhone. Everything else in the app works flawlessly, except that when you select an article to read, the 4 buttons at the bottom are [unlabelled](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW4), so are only announced as "button".

### Steps to reproduce/test case

1. Turn on Voiceover.
2. Tap once on any of the 4 buttons at the bottom when reading an article. It gets read as "button".

**Comment by bourvill** [comment]
this change will be available in next release.
Thanks
